src/doc/man/includes/options-keep-going.md

```diff
@@ -1,5 +1,10 @@
 {{#option "`--keep-going`"}}
 Build as many crates in the dependency graph as possible, rather than aborting
-the build on the first one that fails to build. Unstable, requires
-`-Zunstable-options`.
+the build on the first one that fails to build.
+
+For example if the current package depends on dependencies `fails` and `works`,
+one of which fails to build, `cargo check -j1` may or may not build the one that
```

Comment:
I was looking for an interpolation to use here instead of "check". `cargo {{lower actionverb}} -j1` isn't quite right because it would render as "cargo document" instead of "cargo doc", for example. Do I need to add a new `{{*set` for this or is there one I missed?

Reply:
Sounds reasonable to me. Maybe `{{*set command="check"}}` and it could possibley be reused elsewhere.

Reply:
https://github.com/rust-lang/cargo/pull/12570